pkg/oc/bootstrap/docker/openshift/servicecatalog.go

```diff
@@ -98,6 +101,44 @@ func (h *Helper) InstallServiceCatalog(f *clientcmd.Factory, configDir, publicMa
 		return errors.NewError("cannot instantiate service catalog template").WithCause(err)
 	}
 
+	// once the TSB is created, find the service account it created for the SC to use when calling
+	// the TSB and look up the token associated w/ the SA.
+	var token []byte
+	err = wait.Poll(1*time.Second, 1*time.Minute, func() (bool, error) {
```

Comment:
This looks like an indication that we have some problem with templates that prevents us from creating the SA, getting a predictable secret, and then creating the corresponding role and binding.  Why not make that controller first and make life easier for all installers?

Reply:
> This looks like an indication that we have some problem with templates that prevents us from creating the SA, getting a predictable secret, and then creating the corresponding role and binding. Why not make that controller first and make life easier for all installers?

@bparees see controller pull here: https://github.com/openshift/origin/pull/16029